scintellometry/meta/observations.py

```diff
@@ -61,17 +60,27 @@ def gmrt_rawfiles(file_fmt, fnbase, **kwargs):
     return a 2-tuple for GMRT observation 'key':
     (timestamp file, [file1, ...])
 
-    Number of files depends on whether there is a ``nodes`` keyword argument
+    Number of files depends on whether there is a ``feeds`` keyword argument
     """
-    nodes = kwargs.get('nodes', None)
```

Comment:
I would not remove the `nodes` option, since that would break older `observations.conf` files. Instead, only interpret feeds if nodes is `None`.